src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsError(false);
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsError(true))
```

Comment:
Consider providing a more detailed error message in the `catch` block to help with debugging. For example, you could log the error or display a specific message based on the error type.